src/Commands/Modules/Moderation/EmoteCommand.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Discord;
+using Discord.Commands;
+
+namespace BrackeysBot.Commands
+{
+    public partial class ModerationModule : BrackeysBotModule
+    {
+        [Command("limitemote")]
+        [Summary("List the emote limits of the current channel")]
+        [Remarks("limitemote")]

```

Comment:
We can omit the `Remarks` if `HideFromHelp` is present.

Reply:
Removed.